src/utils/time.ts

```diff
@@ -347,5 +247,3 @@ export const getDateHoursAndMinutes = (
 ): string => format(dateOrTimestamp, DATEFNS_INPUT_TIME_FORMAT);
 
 export const currentMilliseconds = () => new Date().getTime();
-export const currentIso = () => new Date().toISOString();
-export const currentUtc = () => new Date().toUTCString();
```

Comment:
I added these 3 so that people wouldn't type `new Date()` all the time because of it's clunky interface. I think even if unused now, they're good alternative to constantly re-inventing the same when the need arises.